src/server/devHolocronCDN.js

```diff
@@ -17,19 +17,21 @@
 /* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true }] */
 
 import path from 'path';
-
+import fs from 'fs';
 import express from 'express';
 import cors from 'cors';
 import { argv } from 'yargs';
 import oneAppDevCdn from '@americanexpress/one-app-dev-cdn';
 
+const hasLocalModuleMap = () => fs.existsSync(path.join(process.cwd(), 'static', 'module-map.json'));
+
 const app = express();
 
 app.use(cors());
 app.use('/static', oneAppDevCdn({
   localDevPublicPath: path.join(__dirname, '../../static'),
   remoteModuleMapUrl: argv.moduleMapUrl,
-  useLocalModules: true,
+  useLocalModules: hasLocalModuleMap(),
```

Comment:
Should we have a check to see if both `moduleMapUrl` and `localModuleMap` are missing? Or will `moduleMapUrl` always be defined (`http://localhost:3001` IIRC)?

Reply:
I believe it is always defined as you don't have to provide that normally.

Reply:
moduleMapUrl is required if no locally served modules. this check is done in both oneAppDevCdn and argv.js
https://github.com/americanexpress/one-app/blob/main/src/server/config/env/argv.js#L60